assets/lang/en_US.ini

```diff
@@ -539,13 +521,11 @@ FullScreen = Fullscreen
 Hack Settings = Hack settings (may cause glitches)
 Hardware Tessellation = Hardware tessellation
 Hardware Transform = Hardware transform
-hardware transform error - falling back to software = Hardware transform error, falling back to software
 HardwareTessellation Tip = Uses hardware to make curves
 High = High
 Hybrid = Hybrid
 Hybrid + Bicubic = Hybrid + Bicubic
 Ignore camera notch when centering = Ignore camera notch when centering
-Internal Resolution = Internal resolution
```

Comment:
Used in FramebufferManagerCommon:
https://github.com/hrydgard/ppsspp/blob/287dd66f20f2a03acd316a6f5d5b98ff2eb6ad26/GPU/Common/FramebufferManagerCommon.cpp#L2001

-[Unknown]